src/index.ts

```diff
@@ -123,7 +123,7 @@ const resolvers = {
       return types.getMapFor<Campus>("location-units/campuses", { key: "loccamps", cache: context.typeCache }).then(map => map.get(parent.campusId))
     }
   },
-  Date: DateResolver,
+  Date: DateTimeResolver,
```

Comment:
Would it be clearer if we made this a `DateTime` type and updated the schema to use it?

Reply:
@cbeer do you mean replaced Date with DateTime all ~34 times it occurs in `schema.graphql` ? sure !

Reply:
If they're all DateTime values, yep.

Reply:
@cbeer OK this should be good. I checked the docs / responses in Apollo and they all seem good with DateTime.